rpcs3/rpcs3qt/main_window.ui

```diff
@@ -588,6 +589,17 @@
     <string>Configure Auto Pause</string>
    </property>
   </action>
+  <action name="exitAnchorAndLocateLogAct">
+   <property name="text">
+    <string>Exit, Locate And Anchor Log</string>
```

Comment:
If I don't understand what that means, how is the target smoothbrain supposed to understand it?

Reply:
Now called "Exit And Save Log"